hack/.golint_failures

```diff
@@ -107,6 +107,8 @@ pkg/controller/volume/events
 pkg/controller/volume/expand
 pkg/controller/volume/persistentvolume
 pkg/controller/volume/persistentvolume/options
+pkg/controller/volume/persistentvolume/testing
+pkg/controller/volume/scheduling
```

Comment:
Code of testing and scheduling package are extracted from `pkg/controller/volume/persistentvolume` and have some lint failures. I didn't fix them because I don't want make this PR too complicated. I can fix them in a follow up PR.

Reply:
follow up issue: https://github.com/kubernetes/kubernetes/issues/77084